src/components/modal/modal.css

```diff
@@ -0,0 +1,41 @@
+.modal-background {
+  z-index: 1500;
```

Comment:
I would suggest to use z-index values incrementally. You might face issues if you try to add z-index in some other component.

Reply:
Ok, I will find a way around it.